ios/web_view/headers.gni

```diff
@@ -0,0 +1,69 @@
+# Copyright (c) 2024 The Brave Authors. All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at https://mozilla.org/MPL/2.0/.
+
+# This file exposes the //ios/web_view embedder as a regular source_set as well
+# as any additions found in //brave/ios/web_view/public
+
+ios_web_view_public_headers = [
+  "//brave/ios/web_view/public/cwv_ssl_status_extras.h",
+  "//brave/ios/web_view/public/cwv_x509_certificate_extras.h",
+  "//brave/ios/web_view/public/cwv_web_view_extras.h",
+
```

Comment:
empty line?

Reply:
To separate the //brave specific extras and the headers coming from Chromium's //ios/web_view